netclient.rst

```diff
@@ -54,8 +54,8 @@ Debian Distros (debian/ubuntu/mint/pop-os)
 
 .. code-block::
 
-  curl -sL 'https://apt.netmaker.org/gpg.key' | sudo tee /etc/apt/trusted.gpg.d/netclient.asc
-  curl -sL 'https://apt.netmaker.org/debian.deb.txt' | sudo tee /etc/apt/sources.list.d/netclient.list
+  curl https://apt.netmaker.org/gpg.key | gpg --dearmor -o /usr/share/keyrings/netmaker-keyring.gpg
```

Comment:
need sudo for the gpg command. change to `sudo gpg --dearmor -o /usr/share/keyrings/netmaker-keyring.gpg` after the pipe